python/mxnet/contrib/onnx/mx2onnx/_op_translations.py

```diff
@@ -3023,3 +3031,22 @@ def convert_contrib_box_decode(node, **kwargs):
     ]
 
     return nodes
+
+@mx_op.register("_contrib_AdaptiveAvgPooling2D")
+def convert_contrib_AdaptiveAvgPooling2D(node, **kwargs):
+    """Map MXNet's _contrib_BilinearResize2D operator attributes to onnx's operator.
+    """
+    from onnx.helper import make_node
+    name, input_nodes, attrs = get_inputs(node, kwargs)
+
+    output_size = attrs.get('output_size', '1')
+    output_size = convert_string_to_list(output_size)
+
+    if len(output_size) <= 2:
```

Comment:
Why do we need to check this?

Reply:
Yes, when `len(output_size) > 2`, mxnet will perform the calculation with ` output_size = 1`. This check is to handle that.